dll/win32/msgina/shutdown.c

```diff
@@ -324,6 +441,101 @@ LoadShutdownSelState(VOID)
     return dwValue;
 }
 
+VOID
+AddPrefixToStaticTexts(
+    HWND hDlg,
+    BOOL bIsSleepButtonReplaced)
+{
+    WCHAR szBuffer[30];
+
+    for (int i = 0; i < 4; i++)
+    {
+        GetDlgItemTextW(hDlg, IDC_BUTTON_HIBERNATE + i, szBuffer, _countof(szBuffer));
+        SetDlgItemTextW(hDlg, IDC_HIBERNATE_STATIC + i, szBuffer);
+    }
+
+    if (bIsSleepButtonReplaced)
+    {
+        GetDlgItemTextW(hDlg, IDC_BUTTON_HIBERNATE, szBuffer, _countof(szBuffer));
+        SetDlgItemTextW(hDlg, IDC_SLEEP_STATIC, szBuffer);
+    }
+}
+
+VOID
+CreateToolTipForButtons(
+    int controlID,
+    int detailID,
+    IN HWND hDlg,
+    int titleID,
```

Comment:
You either use the the IN/OUT parameter calling convention for every parameter argument of every function or you don't.